### PR TITLE
Documentation about Akkoma and YNH

### PR DESCRIPTION
**Pull request by bugsysop**
In french and english

### CODE REVIEW

File: doc/yunohost.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+| Configuration file          | `/etc/akkoma/config.exs`   | `/etc/akkoma/config.exs`                |
+| Command line path           | `/bin/pleroma_ctl`         | `/var/www/akkoma/live/bin/pleroma_ctl`  |
+
+## Things to remember
+
+- The installation of Akkoma with Yunohost is a so-called OTP installation (and not MIX, i.e. from the sources). Have this in mind, as the administration commands are not the same in either case.
+
+- Akkoma is a _fork_ (divergence) of Pleroma, so the administration command is indeed `pleroma_ctl` (this is not a typo).
+
+- Administration commands in CLI must _necessarily_ be prefixed with `sudo` otherwise it doesn't work, as the `akkoma` system user does not have direct access to the shell.
```
Comment (lapineige) [inline doc/yunohost.md:18]: Please change it to `sudo -u akkoma MIX_ENV=prod` as it will be mandatory to use this, instead of running it as root.

**Review by lapineige — COMMENTED** [review]
While not in the disclaimer section ?

**Comment by bugsysop** [comment]
I renoved the "sudo warning" for now...

**Comment by lapineige** [comment]
I believe it was still good, but anyway, merging, thanks a lot :)

**Pull request merged**